Assets/Editor/UnitTests/Models/FurnitureWorkshopTest.cs

```diff
@@ -57,18 +56,22 @@ public void TestSerialization()
             Idle = "idle",
             Running = "running"
         };
-
-        FileStream fs = new FileStream("Workshop.xml", FileMode.Create);
-        TextWriter writer = new StreamWriter(fs, new UTF8Encoding());
-
+        
+        // serialize
+        StringWriter writer = new StringWriter();
         XmlSerializer serializer = new XmlSerializer(typeof(FurnitureWorkshop));
-        serializer.Serialize(writer, fi);
-        writer.Close();
+        serializer.Serialize(writer, fi);        
 
-        FileStream fr = new FileStream("Workshop.xml", FileMode.Open);
-        var dfi = (FurnitureWorkshop)serializer.Deserialize(fr);
+        StringReader sr = new StringReader(writer.ToString());
 
+        // if you want to dump file to disk for visual check, uncomment this
+        ////File.WriteAllText("Workshop.xml", writer.ToString());
+        
+        // deserialize
+        var dfi = (FurnitureWorkshop)serializer.Deserialize(sr);
```

Comment:
The only tweak I would make is to eliminate this use of var to be in keeping with the style guide. Otherwise this is merge ready.

In fact I'll just make the change myself now...